src/fabric/src/fabric2_server.erl

```diff
@@ -128,3 +133,17 @@ fdb_directory() ->
         Dir ->
             Dir
     end.
+
+fdb_cluster() ->
```

Comment:
Super minor nit, but we can dry out the fdb_directory/fdb_cluster if we just pass the `?FDB_DIRECTORY` or `?FDB_CLUSTER` to a common function.